DOCKER-README.md

```diff
@@ -0,0 +1,44 @@
+Portia
+======
+
+Portia is a tool that allows you to visually scrape websites without any programming knowledge required. With Portia you can annotate a web page to identify the data you wish to extract, and Portia will understand based on these annotations how to scrape data from similar pages.
+
+# Running Portia
+
+The easiest way to run Portia is using Docker.
+
+Clone the repository:
+
+    git clone https://github.com/scrapinghub/portia
+
+Then inside the Portia directory, run:
+
+    docker build -t portia .
+
+Create a Docker [volume](https://docs.docker.com/engine/userguide/dockervolumes/) for the Portia project data:
+
+    docker create --name portia-data -v /app/slyd/data:rw portia
```

Comment:
@e6 when i run this i get a `Error response from daemon: Invalid bind mount spec "/app/slyd/data:rw": volumeinvalid: Invalid volume specification: '/app/slyd/data:rw'` with `Docker version 1.10.0, build 590d5108` in Ubuntu 14.04 LTS